packages/@expo/cli/src/run/resolveBundlerProps.ts

```diff
@@ -38,9 +40,20 @@ export async function resolveBundlerPropsAsync(
     port = 8081;
   }
   Log.debug(`Resolved port: ${port}, start dev server: ${options.bundler}`);
+  let host: string | undefined;
+  try {
+    const response = await fetch(`http://localhost:${port}`, {
+      headers: { 'expo-platform': options.platform ?? 'ios' },
+    });
+    const manifest = await response.json();
```

Comment:
This is pretty heavy. The CLI should probably not be calling into itself in a method that simply coerces props.

Reply:
Yeah, I agree that it's not optimal, but how can we detect that the running bundler instance is being tunneled?

Reply:
@byCedric any thoughts on this?